### DISCUSSION TITLE
Can you populate relation in component?

### DISCUSSION THREAD

**Discussion by Limonische** [strapi/strapi#6840] [Legacy Topics]
So, for example, i have a dynamic zone. Inside of dynamic zone i can choose between some fields, all of which are one-to-one relations. When i fetch the data, only the id of related model is fetched. Is there any way to populate it with all the relation data?

**Comment by abiturma** [comment]
I have the same problem. Could you figure out how to solve it?

**Reply by Limonische** [reply]
Nope, no luck for now.

**Comment by akhilmhdh** [comment]
@Limonische  @abiturma hey guys did u found the solution. I am also stuck on this

**Reply by Limonische** [reply]
Still no progress.

**Comment by derrickmehaffy** [comment]
Hi guys, I believe this issue is also related to this issue: https://github.com/strapi/strapi/issues/7454

We will be working to find a workaround and we are engaging with our backend engineers about it.
